app/helpers/pregnancies_helper.rb

```diff
@@ -6,4 +6,26 @@ def weeks_options
   def days_options
     (0..6).map { |i| ["#{i} days", i] }
   end
+
+  # ticket 241 recently called criteria:
+  # someone has a call from the current_user
+  # that is less than 8 hours old,
+  # AND they would otherwise be in the call list
+
+  def call_list_pregnancies
```

Comment:
It might make more sense to make these model methods and test them as model methods -- what do you think? Like, this might be a great `User` method.